programs/vote/src/vote_state/mod.rs

```diff
@@ -499,10 +508,11 @@ fn check_slots_are_valid(
 // popped off.
 pub fn process_new_vote_state(
     vote_state: &mut VoteState,
-    new_state: VecDeque<Lockout>,
+    mut new_state: VecDeque<LandedVote>,
```

Comment:
prefer to retain these API changes in the timely vote credits PR.  the vote state change PR can do just in time marshaling

Reply:
Seems to me that it would be simpler to introduce new types and vote account changes in a single change.  The extra field of LandedVote doesn't have to be used.  Unless there is a belief that the mere existence of a named field within the code is more risky than special cased serialization logic, but I don't see why that would be true.